src/snowcli/cli/object/manager.py

```diff
@@ -16,9 +14,12 @@ def _get_object_names(object_type: str) -> ObjectNames:
 
 
 class ObjectManager(SqlExecutionMixin):
-    def show(self, *, object_type: str, like: str) -> SnowflakeCursor:
+    def show(
+        self, *, object_type: str, like: str | None = None, **kwargs
+    ) -> SnowflakeCursor:
         object_name = _get_object_names(object_type).sf_plural_name
-        return self._execute_query(f"show {object_name} like '{like}'")
+        like = like or "%%"
```

Comment:
Why these percentages are added now and were not added before?

Reply:
Let me fix this implementation. This was only used in command context and the LikeOption default is `%%` so basically I removed need for passing `like` option if you don't want to use it. outside command context.